packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -68,11 +69,22 @@ class CupertinoDialog extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return new Center(
-      child: new Container(
-        margin: const EdgeInsets.all(10.0),
-        width: _kCupertinoDialogWidth,
-        decoration: _kCupertinoDialogDecoration,
-        child: child,
+      child: new ClipRRect(
```

Comment:
ah interesting. I was trying it before but I didn't understand how it worked. I think the modal barrier causes a scoping problem.